From f1369c8a098508296d4c28b97e1f196d94b7b506 Mon Sep 17 00:00:00 2001
From: Shahab Vahedi <shahab@synopsys.com>
Date: Tue, 22 Dec 2020 12:27:00 +0100
Subject: [PATCH 12/20] arc: Make variable name in comments uppercase

The word "regnum" in comments should be uppercase, because it
reflects a variable name in the code.

gdb/ChangeLog

	* arc-linux-tdep.c: Replace "regnum" with "REGNUM" in comments.

Will be a part of GDB 11:
https://sourceware.org/git?p=binutils-gdb.git;a=commit;h=acf10cacc6bd596ef7327063038bb1ee020c07d0
---
 gdb/ChangeLog        |    4 ++++
 gdb/arc-linux-tdep.c |    4 ++--
 2 files changed, 6 insertions(+), 2 deletions(-)

--- a/gdb/ChangeLog
+++ b/gdb/ChangeLog
@@ -1,3 +1,7 @@
+2020-12-22  Shahab Vahedi  <shahab@synopsys.com>
+
+	* arc-linux-tdep.c: Replace "regnum" with "REGNUM" in comments.
+
 2020-12-22  Anton Kolesov  <anton.kolesov@synopsys.com>
 
 	* Makefile.in (ALLDEPFILES): Add arc-linux-nat.c.
--- a/gdb/arc-linux-tdep.c
+++ b/gdb/arc-linux-tdep.c
@@ -557,7 +557,7 @@
 
   const bfd_byte *buf = (const bfd_byte *) gregs;
 
-  /* regnum == -1 means writing all the registers.  */
+  /* REGNUM == -1 means writing all the registers.  */
   if (regnum == -1)
     for (int reg = 0; reg <= ARC_LAST_REGNUM; reg++)
       supply_register (regcache, reg, buf);
@@ -621,7 +621,7 @@
   gdb_byte *buf = (gdb_byte *) gregs;
   struct gdbarch *gdbarch = regcache->arch ();
 
-  /* regnum == -1 means writing all the registers.  */
+  /* REGNUM == -1 means writing all the registers.  */
   if (regnum == -1)
     for (int reg = 0; reg <= ARC_LAST_REGNUM; reg++)
       collect_register (regcache, gdbarch, reg, buf);
